sdk/daml-lf/ide-ledger/src/main/scala/com/digitalasset/daml/lf/script/IdeLedgerRunner.scala

```diff
@@ -353,7 +353,12 @@ private[lf] object IdeLedgerRunner {
         case SResult.SResultFinal(resultValue) =>
           ledgerMachine.finish match {
             case Right(Speedy.UpdateMachine.Result(tx, locationInfo, _, _, _)) =>
-              ledger.commit(committers, readAs, location, enrich(tx), locationInfo) match {
+              val suffix = Bytes.fromByteArray(Array(0, 0))
+              val enrichedTx = enrich(tx).suffixCid(_ => suffix, _ => suffix) match {
```

Comment:
@andreaslochbihler-da for the record, we are now suffixing contract IDs in the IDE ledger. 
This may need to be reviewed in the context of V2 IDs. To avoid blocking @adrien-piquerez-da we will postpone that for latter.